### ISSUE TITLE
Cypress fails to install properly on Circle CI

### ISSUE THREAD

**Issue by joelhawksley** [cypress-io/cypress#70]
```
cd "client"
npm run ci

> ***@0.0.0 ci /home/ubuntu/***/client
> cypress ci

Cypress was not found: Installing a fresh copy.

  - Downloading Cypress [] 0% 0.0s  - Downloading Cypress [] 2% 0.0s  - Downloading Cypress [] 4% 2.4s  - Downloading Cypress [] 7% 2.7s  - Downloading Cypress [] 9% 3.0s  - Downloading Cypress [] 11% 3.2s  - Downloading Cypress [] 14% 3.1s  - Downloading Cypress [] 16% 3.1s  - Downloading Cypress [] 19% 3.0s  - Downloading Cypress [] 21% 3.0s  - Downloading Cypress [] 24% 2.9s  - Downloading Cypress [] 26% 2.8s  - Downloading Cypress [] 29% 2.7s  - Downloading Cypress [] 31% 2.7s  - Downloading Cypress [] 34% 2.5s  - Downloading Cypress [] 36% 2.5s  - Downloading Cypress [] 39% 2.3s  - Downloading Cypress [] 41% 2.3s  - Downloading Cypress [] 44% 2.2s  - Downloading Cypress [] 47% 2.0s  - Downloading Cypress [] 49% 2.0s  - Downloading Cypress [] 52% 1.8s  - Downloading Cypress [] 54% 1.8s  - Downloading Cypress [] 57% 1.7s  - Downloading Cypress [] 59% 1.6s  - Downloading Cypress [] 62% 1.5s  - Downloading Cypress [] 64% 1.4s  - Downloading Cypress [] 67% 1.3s  - Downloading Cypress [] 70% 1.2s  - Downloading Cypress [] 72% 1.1s  - Downloading Cypress [] 75% 1.0s  - Downloading Cypress [] 77% 0.9s  - Downloading Cypress [] 80% 0.8s  - Downloading Cypress [] 82% 0.7s  - Downloading Cypress [] 85% 0.6s  - Downloading Cypress [] 88% 0.5s  - Downloading Cypress [] 90% 0.4s  - Downloading Cypress [] 93% 0.3s  - Downloading Cypress [] 95% 0.2s  - Downloading Cypress [] 98% 0.1s  - Downloading Cypress [] 100% 0.0s
  - Unzipping Cypress   [] 0% 0.0s  - Unzipping Cypress   [] 0% 21.7s  - Unzipping Cypress   [] 0% 19.3s  - Unzipping Cypress   [] 0% 54.2s  - Unzipping Cypress   [] 0% 54.9s  - Unzipping Cypress   [] 0% 50.6s  - Finished Installing /home/ubuntu/.cypress/Cypress
  - Unzipping Cypress   [] 0% 59.8s
 -Error- 
The Cypress App could not be found.
Expected the app to be found here: /home/ubuntu/.cypress/Cypress

To fix this do (one) of the following:
1. Reinstall Cypress with: cypress install
2. If Cypress is stored in another location, move it to the expected location


npm ERR! Linux 3.14.28-031428-generic
npm ERR! argv "node" "/home/ubuntu/nvm/versions/node/v0.12.7/bin/npm" "run" "ci"
npm ERR! node v0.12.7
npm ERR! npm  v2.13.5
npm ERR! code ELIFECYCLE
npm ERR! ***@0.0.0 ci: `cypress ci`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ***@0.0.0 ci script 'cypress ci'.
npm ERR! This is most likely a problem with the *** package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cypress ci
npm ERR! You can get their info via:
npm ERR!     npm owner ls ***
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/***/client/npm-debug.log

cd "client"
npm run ci
returned exit code 1
```

**Comment by brian-mann** [comment]
Worked with this issue outside of github.

So far the problem seems to be that the `cypress ci` command (which calls into `install`) is failing.

By separating out this into two steps it works:

```bash
cypress install
cypress ci
```

This seems to be a problem with the [Cypress CLI](https://github.com/cypress-io/cypress-cli) tool and not related to the Cypress App itself.

Still investigating.

**Comment by brian-mann** [comment]
Okay this is an issue with node `0.12.7`. Will issue a fix today.

https://github.com/cypress-io/cypress-cli/issues/3

**Comment by brian-mann** [comment]
Fixed in `0.7.3` of the CLI.
